.github/workflows/release.yml

```diff
@@ -23,7 +27,7 @@ jobs:
         VERSION: ${{ steps.tagName.outputs.tag }}
       run: |
           mkdir -p /tmp/out
-          sed -i -e "s,{{VERSION}},$VERSION," cert-manager-issuers-letsencrypt/Chart.yaml
+          sed -i -e "s/appVersion: $PARTITION_COLUMN.*/appVersion: $VERSION/" cert-manager-issuers-letsencrypt/Chart.yaml
```

Comment:
What does this do?

Reply:
I wanted to keep any version number to be able to validate formatting, so in Chart.yaml I need to have some version.
But indeed I committed something extra... it should be
sed -i -e "s/appVersion:.*/appVersion: $VERSION/" cert-manager-issuers-letsencrypt/Chart.yaml